### PR TITLE
Stano/source position

### PR DESCRIPTION
**Pull request by netman92**

### CODE REVIEW

**Comment by sils1297** [comment]
ah can you add a commit before yours adding the setting to the coafile just like we did with the coala-bears coafile?

**Comment by sils1297** [comment]
that should fix the build

File: tests/results/SourceRangeTest.py [hunk 1]
```diff
@@ -86,6 +86,31 @@ def test_json(self):
...
+                        SourcePosition("test_file", 1, 20))
+        self.assertIn(a, b)
+
+        a = SourceRange(SourcePosition("test_file", 1, 2),
+                        SourcePosition("test_file", 2, 20))
+        b = SourceRange(SourcePosition("test_file", 1, 1),
+                        SourcePosition("test_file", 2, 20))
+        self.assertIn(a, b)
+
+        a = SourceRange(SourcePosition("test_file", 1, 2),
```
Comment (AbdealiJK) [inline tests/results/SourceRangeTest.py]: We have an easier way of doing this using `SourceRange.from_values()`
Would make the test easier to read

**Comment by sils1297** [comment]
https://github.com/coala-analyzer/coala/pull/2710/commits/b5638c2f6f2bc67aa56ad55e5ae5ba2b9cf02fde is ready

File: coalib/results/SourceRange.py [hunk 2]
```diff
@@ -125,3 +125,7 @@ def __json__(self, use_relpath=False):
         if use_relpath:
             _dict['file'] = relpath(_dict['file'])
         return _dict
+
+    def __contains__(self, item):
+        file_condition = self.start.file == item.start.file
```
Comment (sils1297) [inline coalib/results/SourceRange.py]: is there any reason for having this in an extra variable and not anding directly? The variable name doesn't really add ckarity for me.

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2710/commits/9024da47cc380344837838fc13bf1bff8a4b943e

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2710/commits/1475410a647244af513b1be10d7e676293d4d37e

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
damn it was me who acknowledged this despite the commit message having a lower case letter after the colon... I need to do better reviews

**Pull request merged**